ingestion/src/metadata/cmd.py

```diff
@@ -198,15 +198,26 @@ def report(config: str) -> None:
     required=False,
 )
 @click.option("--reset-db", help="Reset OpenMetadata Data", is_flag=True)
+@click.option("--ingest-data", help="Start data ingestion", is_flag=True)
```

Comment:
Users should know what this command does, can you add a better description?

Reply:
The option should be skip sample data to avoid breaking the existing workflows. Today we ingest sample data by default and we don't want to break that assumption for existing users. You can also take a look at this PR - https://github.com/open-metadata/OpenMetadata/pull/3496/files#diff-2b38722110cfd94ca38c333ca42feb6387e283888b5dda34081e7421c83fe925R200